### ISSUE TITLE
Origin 3.7.0 installation fails due to origin-clients package

### ISSUE THREAD

**Issue by victorvoronin** [openshift/origin#19035]
##### Version
Try to install 3.7.0 version

##### Steps To Reproduce
1. Step 1
Pointed out 3.7.0 version in inventory using values:
[OSEv3:vars]
openshift_deployment_type=origin
openshift_release=v3.7
openshift_pkg_version=-3.7.0

2. Ran installation

##### Current Result
     Play:     Create OpenShift certificates for master hosts
     Task:     Install the base package for admin tooling
     Message:  Error: Package: origin-3.7.0-1.0.7ed6862.x86_64 (centos-openshift-origin)
                          Requires: origin-clients = 3.7.0-1.0.7ed6862
                          Installed: origin-clients-3.7.1-1.el7.git.0.0a2d6a1.x86_64 (@centos-openshift-origin)
                              origin-clients = 3.7.1-1.el7.git.0.0a2d6a1
                          Available: origin-clients-3.6.0-1.0.c4dd4cf.x86_64 (centos-openshift-origin)
                              origin-clients = 3.6.0-1.0.c4dd4cf
                          Available: origin-clients-3.6.1-1.0.008f2d5.x86_64 (centos-openshift-origin)
                              origin-clients = 3.6.1-1.0.008f2d5
                          Available: origin-clients-3.7.0-1.0.7ed6862.x86_64 (centos-openshift-origin)
                              origin-clients = 3.7.0-1.0.7ed6862

##### Expected Result
Installed Origin 3.7.0 cluster

##### Additional Information
I think a mistake is in this place:
roles/openshift_cli/tasks/main.yaml

************************************************************************************
- name: Install clients
  package: name={{ openshift.common.service_type }}-clients state=present
  when: not openshift.common.is_containerized | bool
************************************************************************************
This piece of code installs latest origin-clients instead of use openshift_pkg_version value

**Comment by jwforres** [comment]
sounds like potentially a duplicate of https://github.com/openshift/origin/issues/19013

@openshift/sig-cluster-lifecycle

**Comment by sdodson** [comment]
Yes, this is a dupe.

**Comment by victorvoronin** [comment]
I don't think it is a dupe.
Will it be possible to install version 3.7.0, using the piece of code that I have posted, after the problems with 3.7.1 packages will be solved. I think no because we will always have latest version using this code. So the problem will not be solved after fixing 3.7.1 issues.
